libs/shared/db/type-cacheable/jest.config.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
```

Comment:
Same Q's as above re: copyright and disabling ESLint globally for the file.

Reply:
Updated and answered above :)